test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointSmokeTest.java

```diff
@@ -811,6 +812,77 @@ public void checkpointBatchSizeTest() throws Exception {
         assertThat(((CheckpointEntry) cpEntry).getSmrEntries().getUpdates().size()).isEqualTo(23);
     }
 
+    @Test
+    @SuppressWarnings("checkstyle:magicnumber")
+    public void checkpointUnompressedSizeTest() throws Exception {
```

Comment:
1. Please add a short description for the test.
2. Can we add a test for the scenario where we have a single large transaction which exceeds both uncompressed size and compressed size limits? The code is supposed to checkpoint that data successfully.